--- a/change_color.sh
+++ b/change_color.sh
@@ -119,7 +119,8 @@ SURUPLUS_GRADIENT_ENABLED=$(echo "${SURUPLUS_GRADIENT_ENABLED-False}" | tr '[:up


 echo ":: Copying theme template..."
-cp -R "$root/suru-plus-aspromauros/Suru++-Asprómauros" "$tmp_dir/"
+rsync -a --copy-unsafe-links "$root/suru-plus-aspromauros/Suru++-Asprómauros"* "$tmp_dir/"
+chmod -R u+w "$tmp_dir"
 echo "== Template was copied to $tmp_dir"

 if [ -n "${ICONS_SYMBOLIC_ACTION:-}" ]; then
