diff --git a/change_color.sh b/change_color.sh
index 03c53bc7..0ac3439c 100755
--- a/change_color.sh
+++ b/change_color.sh
@@ -212,9 +212,11 @@ rm -fr "$DEST_PATH"
 mkdir -p "$DEST_PATH"
 echo -e "\nBuilding theme at $DEST_PATH\n"
 cp -r "$SRC_PATH/src/index.theme" "$DEST_PATH"
+chmod -R u+w "$DEST_PATH"
 for FILEPATH in "${PATHLIST[@]}"; do
 	cp -r "$SRC_PATH/$FILEPATH" "$DEST_PATH"
 done
+chmod -R u+w "$DEST_PATH"
 
 
 cd "$DEST_PATH"
